Src/IronPython/Runtime/PythonFunction.cs

```diff
@@ -106,14 +108,15 @@ public PythonFunction(CodeContext context, FunctionCode code, PythonDictionary g
             _compat = CalculatedCachedCompat();
         }
 
-        internal PythonFunction(CodeContext/*!*/ context, FunctionCode funcInfo, object modName, object[] defaults, MutableTuple closure) {
+        internal PythonFunction(CodeContext/*!*/ context, FunctionCode funcInfo, object modName, object[] defaults, MutableTuple closure, PythonDictionary annotations) {
```

Comment:
nit: Maybe put `annotations` before `closure`?

Reply:
Sure, I can do that. It's internal, so it shouldn't affect anything outside.